helpers/apolloClient-server.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  ApolloClient,
+  InMemoryCache,
+  NormalizedCacheObject
+} from '@apollo/client'
+import merge from 'deepmerge'
+import isEqual from 'lodash/isEqual'
+import { SchemaLink } from '@apollo/client/link/schema'
+import { schema } from '../graphql/schema'
+
+let apolloClient: ApolloClient<NormalizedCacheObject>
+const cache = new InMemoryCache()
+
+function createIsomorphLink() {
+  if (typeof window === 'undefined') {
+    return new SchemaLink({ schema })
+  } else {
+    const { HttpLink } = require('@apollo/client/link/http')
+    return new HttpLink({
+      uri: '/api/graphql',
+      credentials: 'same-origin'
+    })
+  }
+}
+
+function createApolloClient() {
+  return new ApolloClient({
+    ssrMode: typeof window === 'undefined',
+    link: createIsomorphLink(),
+    cache
+  })
+}
+
+export function initializeApollo(
+  initialState: NormalizedCacheObject | null = null
```

Comment:
if you look at [how its used](https://github.com/garageScript/c0d3-app/pull/1289/files#diff-127feee063dd438c812f7f1ddfadfe8f2b1c8e2b6a865342a668c0e7075f67c2R220), it seems like we don't need `initialState`